### PR TITLE
feat: Rework details page and make it suitable for mobile devices

### LINKED ISSUE #60: Frontend: Refactor details design

### PR DESCRIPTION
**Pull request by lucas-goldner**
Closes https://github.com/alphahorizonio/consumat.io/issues/60

### CODE REVIEW

**Review by helmrich — CHANGES_REQUESTED** [review]
We already talked about the changes we should make before merging in the meeting. So if you remove the `defaultLayout` prop from the `Layout` component and make minor changes like the addition of a loading spinner to the movie details view we could merge and then continue tweaking/rewriting the detail view and make additions like the watch status, rating functionality, etc. :smile:

File: components/Layout.tsx [hunk 1]
```diff
@@ -4,15 +4,22 @@ import Navbar from "./navigation/Navbar";
 
 type LayoutProps = {
   children: ReactNode;
+  defaultLayout: String;
 };
 
-const Layout = ({ children }: LayoutProps) => {
+const Layout = ({ children, defaultLayout }: LayoutProps) => {
```
Comment (helmrich) [inline components/Layout.tsx]: We should maybe remove this for now as it's not worth having a property that just affects one page in a wrapper component that's related to all pages in my opinion.

**Review by helmrich — CHANGES_REQUESTED** [review]

**Comment by lucas-goldner** [comment]
Hey I made the requested changes:
- Added the spinner to the movie details 
- Removed the defaultLayout and changed the styles so it fits better with the new maxWidth now

**Review by helmrich — APPROVED** [review]
LGTM

**Pull request merged**